.github/dependabot.yml

```diff
@@ -55,7 +55,6 @@ updates:
       day: "sunday"
       time: "09:00" # 9am UTC
     ignore:
-      # TODO(ibeckermayer): Update RDP client to work with new bitflags.
       - dependency-name: bitflags
```

Comment:
You can remove it from the ignore list too.

Reply:
969f057692f613821a36d47c9548761b0ee8b918